src/main/java/com/example/tournamentMaker/excel/FootballCupWriterService.java

```diff
@@ -0,0 +1,209 @@
+package com.example.tournamentMaker.excel;
+
+import com.example.tournamentMaker.constans.Constans;
+import com.example.tournamentMaker.team.Team;
+import com.example.tournamentMaker.team.TeamRepository;
+import com.example.tournamentMaker.tournament.Tournament;
+import com.example.tournamentMaker.tournament.game.Game;
+import com.example.tournamentMaker.tournament.round.Round;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.compress.utils.IOUtils;
+import org.apache.poi.ss.usermodel.*;
+import org.apache.poi.xssf.usermodel.XSSFClientAnchor;
+import org.apache.poi.xssf.usermodel.XSSFDrawing;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.springframework.stereotype.Service;
+
+import java.io.DataOutputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Comparator;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+@Service
+@RequiredArgsConstructor
+class FootballCupWriterService implements ExcelWriteStrategy {
+    private final String filePath = "D:\\tournamentMaker.xlsx";
```

Comment:
to powinno siedziec w propertisach, albo w plikach projektu - nawet lepiej. Bo potem wrzucisz to na gita i co ? ktos pobierze i skad wezmie tego excela ? z dupy ? :)